### ISSUE TITLE
HardCore World crash[Bug]: 

### ISSUE THREAD

**Issue by burning-brightly** [cavecomputing/SWEM-Beta#16]
### What happened?

When trying to create a hardcore world my game crashes. Even after creating it and seeing it in my world list i tried to launch and it still crashed

### What should happen?

The world should open just fine haha

### Mods

just swem version 1.16.5-0.1.0.1

### SWEM Version

1.16.5-0.1.0.1

### Singleplayer or Multiplayer

- [X] Singleplayer
- [ ] Multiplayer
- [ ] Other (Please add a comment to the issue where this is happening)

### OS

Windows

### Relevant log output

```shell
unfortunately it didnt give me a crash repoty but it did say exit code:0
```

**Comment by burning-brightly** [comment]
I opened the world on regular 1.16.5 minecraft and loaded it up, then closed minecraft once again and relaunched with forge and it seems to have fixed the issue

**Comment by MagnusHJensen** [comment]
This is fixed with version: `1.16.5-0.1.0.2-b` and up.
